code/modules/organs/internal/eyes.dm

```diff
@@ -50,7 +50,7 @@
 	owner.update_client_colour()
 
 /obj/item/organ/internal/eyes/proc/get_colourmatrix() //Returns a special colour matrix if the eyes are organic and the mob is colourblind, otherwise it uses the current one.
-	if(!(BP_IS_ROBOTIC(src)) && owner.stats.getPerk(PERK_OBORIN_SYNDROME))
+	if(!(BP_IS_ROBOTIC(src)) && owner.stats.getPerk(PERK_OBORIN_SYNDROME) && owner.is_dead())
```

Comment:
Why is this check here? Wouldn't it mean that `PERK_OBORIN_SYNDROME` only affects dead mobs?

Reply:
this was an error that was inadvertently generated in the previous review.
![image](https://user-images.githubusercontent.com/49106866/86371409-d8439480-bc4e-11ea-9e63-e57dcbeb34c4.png)